samples/csharp_dotnetcore/49.qnamaker-all-features/Dialog/QnAMakerBaseDialog.cs

```diff
@@ -20,17 +22,18 @@ public class QnAMakerBaseDialog : QnAMakerDialog
         public const string DefaultCardTitle = "Did you mean:";
         public const string DefaultCardNoMatchText = "None of the above.";
         public const string DefaultCardNoMatchResponse = "Thanks for the feedback.";
-
         private readonly IBotServices _services;
+        private readonly IConfiguration _config;
 
         /// <summary>
         /// Initializes a new instance of the <see cref="QnAMakerBaseDialog"/> class.
         /// Dialog helper to generate dialogs.
         /// </summary>
         /// <param name="services">Bot Services.</param>
```

Comment:
>  /// <param name="services">Bot Services.</param> [](start = 6, length = 50)

Add new param

Reply:
Added.

---
In reply to: [433454852](https://github.com/microsoft/BotBuilder-Samples/pull/2430#discussion_r433454852) [](ancestors = 433454852)